py/bitbox02/u2fhid/u2fhid.py

```diff
@@ -98,7 +98,7 @@ def write(hid_device: SupportsReadWrite, data: bytes, cmd: int, cid: int) -> Non
             buf = data[idx : idx + min(data_len, USB_REPORT_SIZE - 7)]
             hid_device.write(
                 b"\0"
-                + struct.pack(">IBH", cid, cmd, data_len & 0xFFFF)
+                + struct.pack(">IBH", cid, cmd & 0xFF, data_len & 0xFFFF)
```

Comment:
instead of clamping, maybe check the range of `cmd` in the beginning of the function and raise a ValueError exception?

Reply:
good idea, I was just lazy and did what was already present. Should probably warn for both cid, cmd and data_len if they are to large.

Reply:
fixed and updated comments